c2corg_api/scripts/es/sync.py

```diff
@@ -0,0 +1,113 @@
+from c2corg_api.models import es_sync, document_types, document_locale_types
+from c2corg_api.models.document import Document
+from c2corg_api.models.document_history import DocumentVersion, HistoryMetaData
+from c2corg_api.models.route import Route, ROUTE_TYPE
+from c2corg_api.models.waypoint import WAYPOINT_TYPE
+from c2corg_api.scripts.es.es_batch import ElasticBatch
+from c2corg_api.search import elasticsearch_config, batch_size, \
+    search_documents
+from c2corg_api.views.document import add_load_for_profiles
+from sqlalchemy.orm import joinedload
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def sync_es(session):
+    last_update, date_now = es_sync.get_status(session)
+
+    if not last_update:
+        raise Exception('No last update time, run `fill_index` to do an '
+                        'initial import into ElasticSearch')
+
+    # get all documents that have changed since the last update
+    # TODO also check changes to associations and user names
```

Comment:
I created a new issue for the user names: https://github.com/c2corg/v6_api/issues/244